salt/salt.spec

```diff
@@ -978,8 +982,8 @@ list of active executors.  This package add the configuration file.
 
 %if "%{flavor}" == "testsuite"
 
-%if 0%{?_nosinglespec}
-%package -n %{python_module salt-testsuite}
+%if 0%{?singlespec_compat}
+%package -n python3-salt-testsuite
```

Comment:
Why is this changed back from `%{python_module}`?

Reply:
there are 2 cases of building the package, in case of the system not compatible with singlespec we are choosing `python3-salt-testsuite` which is actually static and `%{python_module salt-testsuite}` is producing exactly `python3-salt-testsuite` anyway, the other option is `python-salt-testsuite` with singlespec which is handled by singlespec macroses and copied the number of flavours accorind to the list.
For this particular case there shouldn't be any other options than `python3-salt-testsuite` so the macro is redundant there.